### ISSUE TITLE
Painted Representation: veränderte Darstellung der Originalfotos

### ISSUE THREAD

**Issue by OKienzler** [saw-leipzig/kucha#356]
z.B. in ID 996 sieht das Bild aus einer Publikation sehr hell aus, tatsächlich sind da kräftige Farben:
![grafik](https://user-images.githubusercontent.com/47562310/171909895-7ae55b7f-29ef-4db6-b477-726b932d3e72.png)

**Comment by erikradisch** [comment]
Aber dafür gibt es doch die Möglichkeit, Filter zu nutzen...

**Comment by OKienzler** [comment]
Weiß gerade nicht was Du meinst? Ich meinte, dass die Bilder auch in der imagepool anders erscheinen als die sollten, z.B.
Kizil080_CeilingRi_Reg02_No01_ZXBY_II_p277_pl247_master (ID 4567)
Kizil171_RARi_OuterWall_ ZXBY_I_p277_pl248_master (ID 857)
Kumtura38_x_ZXBY_IV_p277_pl265_master (ID 13382)

ABer diese ist z.B. richtig dargestellt:
ID 1152
Kizil171_RARi_OuterWall_ ZXBY_I_p277_pl248

**Comment by OKienzler** [comment]
![grafik](https://user-images.githubusercontent.com/47562310/171919085-83189942-4815-4aae-a7a0-7c24e068d247.png)

**Comment by erikradisch** [comment]
Sollten jetzt wieder ordentlich angezeigt werden, Du musst aber den Cache löschen.

**Comment by OKienzler** [comment]
passt noch nicht, es sind jetzt andere Farben aber noch nicht die Originale
![grafik](https://user-images.githubusercontent.com/47562310/174259581-46e149ee-4cdb-4c99-a843-db5178e73a44.png)

**Comment by OKienzler** [comment]
Nee, die ich gesehen habe sind alle aus Publikation, also nur "copyright" Bilder:
Kizil080_CeilingRi_Reg02_No01_ZXBY_II_p277_pl247_master (ID 4567)
Kizil171_RARi_OuterWall_ ZXBY_I_p277_pl248_master (ID 857)
Kumtura38_x_ZXBY_IV_p277_pl265_master (ID 13382)
